### PR TITLE
[DOCFIX] Updating the Logging-Convention.md of the Chinese Version

### PR DESCRIPTION
**Pull request by shizihao123**

### CODE REVIEW

File: docs/cn/Logging-Conventions.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+## 日志记录约定
+
+Alluxio使用如下的日志级别：
+
+错误级别日志
+
+* 错误级别日志表示无法恢复的系统级问题。
+* 错误级别日志总是伴随着堆栈跟踪信息。
+
+告警级别日志
```
Comment (ifcharming) [inline docs/cn/Logging-Conventions.md]: s/告警/警告 ?

File: docs/cn/Logging-Conventions.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+log4j.appender.MASTER_LOGGER_SOCKET=org.apache.log4j.net.SocketAppender
+log4j.appender.MASTER_LOGGER_SOCKET.Port=<PORT>
+log4j.appender.MASTER_LOGGER_SOCKET.RemoteHost=<HOSTNAME_OF_LOG_SERVER>
+log4j.appender.MASTER_LOGGER_SOCKET.ReconnectionDelay=<MILLIS_TO_WAIT_BEFORE_RECONNECTION_ATTEMPT>
+log4j.appender.MASTER_LOGGER_SOCKET.layout=org.apache.log4j.PatternLayout
+log4j.appender.MASTER_LOGGER_SOCKET.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%n
+```
+
+这是Alluxio使用远程日志的一个例子，鼓励用户探索各种由Log4J或第三方提供的appender和配置选项，从而创建最适合实际用例的日志记录解决方案。
```
Comment (ifcharming) [inline docs/cn/Logging-Conventions.md]: nit: 这是Alluxio使用远程日志的一个例子。Alluxio鼓励用户...

File: docs/cn/Logging-Conventions.md (hunk 1 above)
Comment (shizihao123) [inline docs/cn/Logging-Conventions.md]: Thank you for your review , and I will fix that.

File: docs/cn/Logging-Conventions.md (hunk 2 above)
Comment (shizihao123) [inline docs/cn/Logging-Conventions.md]: Thanks a lot , and I will fix that.

**Comment by shizihao123** [comment]
I have fixed the problems , please review again, Many thanks. @ifcharming

**Pull request merged**